src/mgr/Action.cc

```diff
@@ -11,6 +11,8 @@
 #include "squid.h"
 #include "CacheManager.h"
 #include "comm/Connection.h"
+#include "compat/socket.h"
```

Comment:
API provided by this include is not used in this .cc file.
```suggestion
```

Reply:
removed